custom/icds/templates/icds/messaging/indicators/tam/child_vaccinations_complete.txt

```diff
@@ -0,0 +1,3 @@
+{# NOTE: Django assumes templates are stored with UTF-8 encoding by default. Be sure to save this template with UTF-8 encoding. #}
+
+बाधाई हो ! आपने {{ child_name }} को सभी टीके लगवा दिए हैं |
```

Comment:
Hindi?

Reply:
Yep, that is Hindi. I removed it and replaced it with Tamil.

Reply:
@marissahrrsn I must have missed the PR with the updated translation for this one. Can you link me to it?

Reply:
Ignore previous comment. Posted in wrong place.